### PR TITLE
REV-029-refactor-verify-phone-page-file-structure

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
Screenshot from 2022-01-09 09-46-28

### CODE REVIEW

File: src/components/shared/Sidebar/sidebar.styles.js [hunk 1]
```diff
@@ -29,7 +30,11 @@ const fadeOut = keyframes`
 `;
 
 export const MenuItemsContainer = styled.div`
-  animation: ${({ isCollapsed }) => (isCollapsed ? fadeOut : fadeIn)}
+  animation: ${({ isCollapsed }) => {
+      if (isCollapsed === true) return fadeOut;
+      if (isCollapsed === false) return fadeIn;
+      return '';
+    }}
```
Comment (AseelRousan-op) [inline src/components/shared/Sidebar/sidebar.styles.js:37]: can we revert this change here 
```suggestion
animation: ${({ isCollapsed }) => (isCollapsed ? fadeOut : fadeIn)}
```

File: src/components/shared/Sidebar/index.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ import {
 function Sidebar() {
   const { isLoggedIn } = useUser();
 
-  const [isCollapsed, setIsCollapsed] = useState(false);
+  const [isCollapsed, setIsCollapsed] = useState(' ');
```
Comment (AseelRousan-op) [inline src/components/shared/Sidebar/index.jsx:28]: why we're removing the initial false here ?

**Review by AseelRousan-op — CHANGES_REQUESTED** [review]

**Comment by AseelRousan-op** [comment]
@momokdad I talked to Ibrahim and he said that you'll address these changes, Thanks in advance

**Comment by AseelRousan-op** [comment]
Talked to @momokdad I'll mark the above changes as resolved

**Review by AseelRousan-op — APPROVED** [review]

**Review by AhmadAlShwaiki — APPROVED** [review]
Looks good!

**Review by mohdabed123 — APPROVED** [review]
LGTM

**Pull request merged**